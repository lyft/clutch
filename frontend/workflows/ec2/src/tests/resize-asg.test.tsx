import React from "react";
import { shallow } from "enzyme";

import ResizeAutoscalingGroup from "../resize-asg";

describe("Resize Autoscaling Group workflow", () => {
  let component;

  beforeAll(() => {
    component = shallow(
      <ResizeAutoscalingGroup resolverType="clutch.aws.ec2.v1.AutoscalingGroup" />
    );
  });

  it("renders correctly", () => {
    expect(component.debug()).toMatchSnapshot();
  });
});
