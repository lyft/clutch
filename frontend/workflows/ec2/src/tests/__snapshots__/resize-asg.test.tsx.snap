// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resize Autoscaling Group workflow renders correctly 1`] = `
"<Wizard dataLayout={{...}} heading={[undefined]}>
  <GroupIdentifier name=\\"Lookup\\" resolverType=\\"clutch.aws.ec2.v1.AutoscalingGroup\\" />
  <GroupDetails name=\\"Modify\\" />
  <Confirm name=\\"Confirmation\\" notes={{...}} />
</Wizard>"
`;
